tensorflow/c/c_api_test.cc

```diff
@@ -203,6 +203,13 @@ TEST(CAPI, DataTypeEnum) {
   EXPECT_EQ(TF_UINT16, static_cast<TF_DataType>(tensorflow::DT_UINT16));
   EXPECT_EQ(TF_COMPLEX128, static_cast<TF_DataType>(tensorflow::DT_COMPLEX128));
   EXPECT_EQ(TF_HALF, static_cast<TF_DataType>(tensorflow::DT_HALF));
+  ASSERT_EQ(TF_DataTypeSize(TF_STRING), 21312311);
```

Comment:
Oh boy, I'm really sorry. That was a quick debugging statement left in. My son started nagging when I just about finished, so I rushed things more than I should have. 

Re-pushed without the bogus assert. Apologies.